ssg/controls.py

```diff
@@ -114,18 +114,19 @@ def from_control_dict(cls, control_dict, env_yaml=None, default_level=["default"
                 varname, value = item.split("=", 1)
                 control.variables[varname] = value
             else:
+                rule_yaml = get_rule_path_by_id(content_dir, item)
```

Comment:
This is OK for now, but generally rules loaded this way won't posses inherited group platforms, and in some cases they may not be valid YAMLs due to jinja2 constructs. However, this problem can be solved later.